examples/measure.js

```diff
@@ -138,6 +132,10 @@ var map = new ol.Map({
 
 map.on('pointermove', pointerMoveHandler);
 
+map.on('mouseout', function() {
```

Comment:
Did you mean to use `jQuery.mouseleave()` here?  The map doesn't fire a `mouseout` event.